README.md

```diff
@@ -56,6 +56,8 @@ To install a _Score Implementation_, see the installation guide in each reposito
 - [score-compose](https://github.com/score-spec/score-compose#-installation)
 - [score-helm](https://github.com/score-spec/score-helm#-installation)
 - [score-humanitec](https://github.com/score-spec/score-humanitec#-installation)
+- For linux based systems, run this script on your CLI:  
+  "curl -L https://github.com/score-org/score-cli/releases/download/v1.0.0/score-cli_1.0.0_linux_amd64.tar.gz | tar -xz -C /usr/local/bin/" 
```

Comment:
Thanks for looking into a solution for [issue #5](https://github.com/score-spec/spec/issues/5). Two things:

- The script you provided won't run as we're not working with a generic `score-cli` but rather individual CLI tools such as `score-compose` or `score-helm`. This is documented in our knowledge base, see [here](https://docs.score.dev/docs/get-started/install/#step-1-download-the-latest-release).

- As part of [issue #5](https://github.com/score-spec/spec/issues/5) we were looking for more of a generic/automated solution as proposed [here](https://github.com/score-spec/spec/issues/5#issuecomment-1343050102) for example. In the above script, the user is still required to specify the version number. 

Happy to chat about this further and discuss a suitable solution!